Source/Boxed.Mapping/Boxed.Mapping.csproj

```diff
@@ -15,4 +15,8 @@
     <None Include="..\..\Documentation\Boxed.Mapping.md" Pack="true" PackagePath="\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="Factory.Unity.cs" />
```

Comment:
I think you have these the wrong way around. The above file should be .Unity.cs

Reply:
That's a `Remove` tag, it removes the file from the compilation list. Since this is a non-Unity csproj, `Remove="Factory.Unity.cs"` is correct here